SR2020/2019-10-11-kickstart-details-confirmation.md

```diff
@@ -0,0 +1,28 @@
+---
+to: sr2020-teams
+subject: Confirming Kickstart details
+---
+
+Hi all!
+
+Kickstart is only just around the corner, so it's time to confirm details for the kickstart events happening this year.
+
+As stated in a previous email, we're running 3 kickstart events this year:
+
+- The University of Southampton
+- The University of Cambridge
+- The offices of [Thread](https://www.thread.com/), Whitechapel
+
+Event-specific information can be found on the [events pages on our website](https://studentrobotics.org/events/). These are constantly being updated with more information, so be sure to keep an eye out!
+
+You all requested a specific kickstart event when you signed up for the year. We've not had to rearrange things so far, so you've all got places at those venues! If you wish to change venue, or cannot attend kickstart, please let us know, so we can make the appropriate arrangements with your kit.
+
+Which kickstart we're expecting you at: ** Google Sheet Link **
```

Comment:
Might be nice to do a mail merge for this.

Reply:
Google Groups doesn't support mail merge, and I really want *all* comms to go via this mailing list.

The list can double as the public index for teams, which can be used as a reference for volunteers also.

Reply:
> Google Groups doesn't support mail merge, and I really want _all_ comms to go via this mailing list.

Serious question: why? (what value does that provide?)

> The list can double as the public index for teams, which can be used as a reference for volunteers also.

I'm not sure we necessarily _want_ to bind ourselves that all emails to teams be fully public. IMO any information of that nature which we deem important (rather than mundane) should probably be on the website too. Information which is merely mundane likely doesn't need to be public at the point of transmission.

We already have this repo as a reference for volunteers and for the future, so I'm not sure I see the need for another copy of everything on a list?

Combing back to the original concern:

> Google Groups doesn't support mail merge

This strikes me as a good reason _not_ to use Google Groups for this sort of thing. Personalising emails to the recipient isn't something which should be especially hard and it does add a certain level of polish which improves the perception of us as an organisation.

We can probably get away with this for teams who have participated lots before, however we also need to think about this from the perspective of a new team. I'm not saying this specific point is going to be make-or-break for them, more that we should consider what their experience will be and do what we can to ensure that it feels polished.

For inviting teams to Kickstart specifically, I think its important that we are clear in the email about where a team should go to. We should likely also consider including some location specific details, even if it's just a link to the specific page on the website as the first pass of that.

Reply:
Ah, there's some ambiguity here. By "the list", I meant the google sheet of teams and where we're expecting them for kickstart. The teams list archive I agree isn't the right forum for volunteers to find details, as they likely don't have access to it anyway.

I agree a lack of mail merge isn't especially ideal, but it's *very* easy to work around. Lat year we used mail merge exactly twice: once for this very email, and again for confirming addresses for shipping kits. I don't see it as a critical feature

Reply:
I'm not sure why we feel that sending out non-personalised information like this is a viable option. I don't think that SR has ever done this before (at least not to teams), even when we had far more teams.

We have used various tooling to do so (even [writing our own](https://github.com/PeterJCLaw/fritter/) at one point), so it feels very much like a step backwards not to be doing so now.

I think it's important that teams receive clear, direct instructions about which event they should be going to.

Reply:
Correct, and this is something we should probably work on improving in future. But not now.

Reply:
Why not now? Inviting someone to an event with multiple locations and not confirming which one feels wrong to me. Yes they specified a preference when signing up but does everyone remember what there preference was?

If we don't have a technical tool to do this can we manually 'mail merge' this email and work on a tool later? If its a question of taking a long time can we split the emails between members of the competition team so each person only has to send a few emails?

Reply:
We're still telling people explicitly which venue we're expecting them at, the difference is that it's in an attached Google sheet, rather than inline in the email.

I really fail to see what the issue with putting a google sheet link in the email is? They still get exactly the information they need, it's just 1 extra click, which is nothing.

Delaying this email simply because of that feels wrong. A review of Google Groups may be in order, but it was done as the trustees wanted us to move. Seeing as we drive everything through GitHub anyway, an integration with that could definitely be in order!

If this is really all that's blocking send this email, I'm probably going to just send it, and we can work on this issue some other time.